### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone or `ldd`.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+image: ghcr.io/capuanob/znet-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image name.

File: fuzz/fuzz.c [hunk 3]
```diff
@@ -0,0 +1,85 @@
+#define ZN_IMPLEMENTATION
```
Comment (xansec) [inline fuzz/fuzz.c]: Can you change the name of this file to indicate what's being fuzzed?

File: tests/bench_server.c [hunk 4]
```diff
@@ -9,9 +9,10 @@
 /* server */
```
Comment (xansec) [inline tests/bench_server.c]: I presume these changes were necessary to get the fuzz harness running.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: fuzz/fuzz.c (hunk 3 above)
Comment (capuanob) [reply fuzz/fuzz.c:1]: Correct, this file was very much out of date with the current state of the method signatures and wasn't capable of compiling

File: tests/bench_server.c (hunk 4 above)
Comment (capuanob) [reply tests/bench_server.c]: Correct, this file was very much out of date with the current state of the method signatures and wasn't capable of compiling

Comment (capuanob) [reply tests/bench_server.c]: I reset it to its initial state though, since I don't actually use it and just copied it. Good catch

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**